content/en/database_monitoring/setup_sql_server/troubleshooting.md

```diff
@@ -0,0 +1,175 @@
+
+2. If your SQL Server instance does not require encryption to connect (`rds.force_ssl=0` on AWS), then update the connection string to include `Use Encryption for Data=False;`. For example:
+
+```yaml
+# example uses windows authentication
+instances:
+  - host: localhost,1433
+    connection_string: "Trusted_Connection=yes;Use Encryption for Data=False;"
+    connector: adodbapi
+    adoprovider: MSOLEDBSQL19
+```
+
+3. Install the [2018 version of the MSOLEDBSQL driver][8], which does not use encryption by default. In order to do this, update the `adoprovider` to `MSOLEDBSQL`. For example:
+
+```yaml
+# example uses windows authentication
+instances:
+  - host: localhost,1433
+    connection_string: "Trusted_Connection=yes;"
+    connector: adodbapi
+    adoprovider: MSOLEDBSQL
+```
+
+If you are using a driver **other than `MSOLEDBSQL` 2019**, this error can be resolved by setting `TrustServerCertificate=yes` in the connection string. For example, for the 2017 `ODBC` driver:
+
+```yaml
+# this example uses SQL Server authentication
+instances:
+  - host: localhost,1433
+    username: datadog
+    password: <DD_AGENT_PASSWORD>
+    connection_string: "TrustServerCertificate=yes;"
+    connector: odbc
+    driver: '{ODBC Driver 17 for SQL Server}'
+```
+
+#### SQL Server unable to connect 'SSL Security error (18)'
+
+This is a known issue for older versions of the SQL Server ODBC driver. You can check which version of the driver is being used by the agent by looking at the connection string in the error message.
+
+For example, if you see `Provider=SQL Server` in the connection string of the error message, upgrading to a newer version of the ODBC driver will fix the error.
+
+This issue is described in more detail in this [Microsoft blog post][9]
+
+
+#### SQL Server 'Unable to connect: Adaptive Server is unavailable or does not exist'
+
+This error can sometimes just be the result of not properly setting the `host` field. For the integration, you should the `host` field with the following syntax: `host:server,port`.
+
+For example:
+
+```
+host: sqlserver-foo.cfxxae8cilce.us-east-1.rds.amazonaws.com
+```
+Should be set as:
+```
+host: sqlserver-foo.cfxxae8cilce.us-east-1.rds.amazonaws.com,1433
+```
+
+### Other common questions
+
+#### SQL Server user tag is missing on the Query Metrics page
+
+The `user` tag has been removed from all sqlserver metric telemetry.
+
+The `user` tag was previously incorrect and did not reflect the user who executed the query in the Query Metrics UI. This is due to a technical limitation in the tables that we are querying in sqlserver for this data.
+
+#### Why are there so many “CREATE PROCEDURE” queries?
+
+In versions of the agent older than 7.40.0, there exists a bug where `PROCEDURE` statistics are over counted. This leads to seeing many executions of `CREATE PROCEDURE...` in the database-monitoring Query Metrics UI. In order to fix this issue, please upgrade to the latest version of the sqlserver agent.
```

Comment:
probably: `... please upgrade to the latest version of the Datadog agent` at the end